tests/testing_common.py

```diff
@@ -228,6 +232,45 @@ def get_grid_parameters(self, grid_parameters, filter_params_func=None):
         return generated_tests
 
 
+@contextmanager
+def model_cache_context(kind, model_ids):
+    global HUB_MODEL_ACCESSES
+    override = {}
+
+    if isinstance(model_ids, str):
+        model_ids = [model_ids]
```

Comment:
Isn't `model_ids` always a str? I couldn't see a different case. There is `model_id + config_kwargs.get("tokenizer_name_or_path", "")` but that's also a string, right?

Reply:
I thought that it might be possible to have sections where there are several models in the context but so far I didn't find something like that. It also turned out to be useless in its current form for the tokenizer case since the tokenizer default value ("") would still produce a cache miss, even if `model_id` was already hit (-> you'd have to ignore empty entries).

I think I'm going to remove it and make it only accept a string.